docs/installation/initial-setup.mdx

```diff
@@ -129,25 +130,25 @@ DiscordConsoleChannelId: "219559838890459137"
 
 :::info 
 
-Open your server settings by clicking on the arrow to the right of the server name and choose `Server Settings`
+Open your server settings by clicking on the name of your server and choose `Server Settings`
 
-![open server settings](/assets/open_server_settings.png)
+![open server settings](/assets/server_settings.png)
 
 :::
 
 :::info 
 
-Go to the `Roles` tab and create a new role. We've named ours `Bot`. Add the `Administrator` permission (or permissions listed below) to the new role.
+Go to the `Roles` tab and create a new role. We've named ours `Bot`. Add the `Administrator` permission (or [permissions listed below](#server-permissions)) to the new role.
 
-![create bot role](/assets/create_bot_role.png)
+![create bot role](/assets/create_bot_role.gif)
 
 :::
 
 :::info 
 
 Switch to the `Members` tab and add the new `Bot` role to your bot by clicking on the `+` next to the bot's name and choosing the role.
 
-![Add Bot-rule to bot](/assets/add_role.png)
+![Add Bot-rule to bot](/assets/add_role.gif)
```

Comment:
I know this was how it was originally, but this is surely supposed to say bot right?
```suggestion
![Add Bot-role to bot](/assets/add_role.gif)
```